authn/auth_token.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright the Hyperledger Fabric contributors. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+option go_package = "github.com/hyperledger/fabric-protos-go/authn";
+option java_package = "org.hyperledger.fabric.protos.authn";
+
+package authn;
+
+message TokenRequest {
+	// Serialized msp.SerializedIdentity.
```

Comment:
perhaps it makes sense to put here (for optionally) the identity of the peer as well, to prevent this request being transferable across nodes?

Reply:
Will probably end up going with a timestamp